app/src/main/res/menu/choose_passage_book_menu.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/alphabetical_order_opt"
+        android:title="@string/sort_by_alphabetical"
+        android:checkable="true"/>
+
+    <item
+        android:id="@+id/row_order_opt"
+        android:title="Flow by row (LTR)"
```

Comment:
string need to be translatable (see example from the above item)

Reply:
My bad @tuomas2  I focused on other stuff and missed this. Just fixed it but strings files are conflicted and I cannot resolve them

Reply:
merge latest develop branch to your branch and resolve merge conflicts manually?

Reply:
do not edit any translations btw, just English. translations come from transifex and changes to these files will be overwritten

Reply:
Done!